src/osip-simulation-controller/src/main/resources/Bundle.properties

```diff
@@ -14,3 +14,17 @@ opcua.pipe.flowRate=Flow rate
 opcua.motor=Motor
 opcua.motor.rpm=Rotations per minute
 opcua.pipe.in=Incoming pipe
+controller.helpdialog.title=Help
+monitoring.helpdialog.closeButton=Close
+controller.helpdialog.mainTab.header=General
+controller.helpdialog.mainTab.introduction=The OSIP production site simulation simulates a production site consisting of three upper tanks containing liquids, which get mixed through some pipes into a mix tank. It is possible to adjust a motor speed, temperatues and valves.
+controller.helpdialog.mainTab.layoutSection=Layout
+controller.helpdialog.mainTab.layoutText=The simulation has three tanks on top which get liquid from pipes above of them. Each tank has a valve above and below the tank to set the input and output flow rates. Additionally there is a temperature sensor attached to each tank.From each of those tanks a pipe is connected to the mix tank, which mixes the liquids of the upper tanks. The mix tank is a regular tank without a valve above and with a mix motor in it.
+controller.helpdialog.controlTab.header=Control
+controller.helpdialog.controlTab.introduction=The control window can be opened via the "File" menu. The control window is necessary to modify temperatures, valves and the mix motor speed.
+controller.helpdialog.controlTab.generalSection=General
+controller.helpdialog.controlTab.generalText=There are four tabs, one for each tank. The first three tabs, which control the upper tanks, consist of two sliders to set the input and output valve state in percent and a slider to change the input liquid temperature.
+controller.helpdialog.controlTab.mixTankSection=Mix Tank Control
+controller.helpdialog.controlTab.mixTankText= The fourth tab is about the mix tank, and thus doesn't have a slider for the input valve because the mix tank doesn't have an input valve and it doesn't have a temperature slider, as the input temperature is determinted by the output temperature of the upper tanks. There is an output valve slider and additionally to the upper tanks, there is the possibility to select the motor speed of the mix motor.
```

Comment:
The fourth tab is about the mix tank **[Kein Komma]** and thus doesn't have a slider for the input valve because the mix tank doesn't have an input valve **;** and it doesn't have a temperature slider, as the input temperature is **determined** by the output temperature of the upper tanks. There is an output valve slider and **,** additionally to the upper tanks, there is the possibility to select the motor speed of the mix motor.